### PR TITLE
STU-1225 add translation 

### PR DESCRIPTION
**Pull request by lucaatstudylydotcom**
add translation

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,2 @@
-node_modules
 coverage
-.idea
\ No newline at end of file
+.idea
```
Comment (MartinSchmidtStudyly) [inline .gitignore:3]: Why have you removed the node_modules? As it has a package.json file it also will have a node_modules file as soon as `npm install` is called.
```suggestion
node_modules
.idea
```

File: lang/contexts.json [hunk 2]
```diff
@@ -0,0 +1,3 @@
+{
```
Comment (MartinSchmidtStudyly) [inline lang/contexts.json:1]: What is this file and why is it needed?

File: lang/translations/en.po [hunk 3]
```diff
@@ -0,0 +1,9 @@
+msgid ""
+msgstr ""
+"Language: \n"
+"Language-Team: \n"
+"Plural-Forms: \n"
```
Comment (MartinSchmidtStudyly) [inline lang/translations/en.po:5]: Why are these properties empty and also why are they only present in the `en.po` file and not in the `de.po` file?

Comment (lucaatstudylydotcom) [reply lang/translations/en.po:5]: these is the standart text for localisation. i will add the rules to the files. see : 
https://ckeditor.com/docs/ckeditor5/latest/framework/guides/deep-dive/ui/localization.html
in english, it is empty, in german there are rules (as seen in the internallink po file), i will put them in the other files too so it is consistent

File: lang/contexts.json (hunk 2 above)
Comment (lucaatstudylydotcom) [reply lang/contexts.json:1]: this file is here to give additional context to the text to translate. added the only text to this

File: .gitignore (hunk 1 above)
Comment (lucaatstudylydotcom) [reply .gitignore:3]: readded

**Review by MartinSchmidtStudyly — APPROVED** [review]

**Pull request merged**